modules/kubernetes-addons/argocd/values.yaml

```diff
@@ -1,3 +1,7 @@
+configs: 
+  secret:
+    argocdServerAdminPassword: ${admin_password}
+
```

Comment:
This can be removed with the above mentioned change

Reply:
Done